src/main/resources/application.yml

```diff
@@ -30,17 +30,30 @@ springdoc:
   swagger-ui:
     path: '/'
     display-request-duration: true
-    operationsSorter: 'method'
+    operationsSorter: 'alpha'
   show-login-endpoint: true
 
 server.port: ${PORT:443}
 
 good-choice:
+  bootstrap:
+    employee-creation:
+      enabled: ${EMPLOYEE_CREATION_BOOTSTRAP_ENABLED:true}
+      accounts:
+        - role: 'hr'
+          email: '${SUPER_HR_EMAIL_ADDRESS:goodchoice.hr.super@gmail.com}'
+          password: '${SUPER_HR_PASSWORD}'
+          first-name: 'HR'
+          last-name: 'Super'
+
+  external-services:
+    web-frontend: '${WEB_FRONTEND_URL:https://good-choice-web.herokuapp.com}'
```

Comment:
Why do we need that?

Reply:
We are sending emails with confirmation links like `$webFrontendUrl/foo?token=buzz`